src/test/functional/codecept.conf.ts

```diff
@@ -20,12 +20,19 @@ export const config: CodeceptJS.MainConfig = {
       enabled: true,
       require: '@codeceptjs/allure-legacy',
     },
-
+    pauseOnFail: {
+      enabled: !testConfig.TestHeadlessBrowser,
+    },
+    retryFailedStep: {
+      enabled: true,
+    },
+    tryTo: {
```

Comment:
can you try removing the tryTo option, it doesn't seem like something we want. Read up on the docs about it.